docs/zh_cn/tutorials/how_to_do_regression_test.md

```diff
@@ -0,0 +1,213 @@
+# 如何进行回归测试
+
+<!-- -->
+这篇教程介绍了如何进行回归测试。部署配置文件由`每个codebase的回归配置文件`，`推理框架配置信息`组成。
+
+<!-- TOC -->
+
+- [如何进行回归测试](#如何进行回归测试)
+    - [1. 用法](#1-用法)
+        - [示例](#示例)
+        - [参数解析](#示例)
+    - [2. 回归测试配置文件](#2-回归测试配置文件)
+        - [示例及参数解析](#示例及参数解析)
+    - [3. 生成的报告](#3-生成的报告)
+        - [模板](#模板)
+        - [示例](#示例)
+    - [4. 支持的后端](#4-支持的后端)
+    - [5. 支持的Codebase及其Metric](#5-支持的Codebase及其Metric)
+    - [6. 注意事项](#7-注意事项)
+    - [7. 常见问题](#8-常见问题)
+
+<!-- TOC -->
+
+## 1. 用法
+
+```shell
+python ./tools/regression_test.py \
+    --deploy-yml "${DEPLOY_YML_PATH}" \
+    --test-type "${TEST_TYPE}" \
+    --backends "${BACKEND}" \
+    --work-dir "${WORK_DIR}" \
+    --device-id "${DEVICE}" \
+    --log-level INFO
+```
+
+### 参数解析
+
+- `--deploy-yml` : 需要测试的 codebase，eg.`configs/mmdet/mmdet_regression_test.yaml`，如果设置为 `all` 即全部测试。
+- `--test-type` : 测试模式：`convert` 测试转换，`precision` 测试精度。
```

Comment:
here we only have two choices, we could you a bool argument such as --test-perf, if it is set to true, then we test performance otherwise, we only test conversion.

Reply:
Done. Set `--performance` to test performance. Otherwise, we only test conversion.